src/codeu/chat/util/encryption/Encryptor.java

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
A general comment similar to what I told Eric as well. If you're writing a class to perform encryption/decryption, it is always better, safer and easier to use any of the standard encryption libraries to perform the encrypt/decrypt operations.

You just supply the secrets, the data and let the library do the rest, instead of reinventing the wheel here. Especially for encryption, please avoid using custom cryptographic routines, as they may not be secure. :)